From: Reiner Herrmann <reiner@reiner-h.de>
Date: Sat, 21 Dec 2019 22:31:04 +0100
Subject: remove operating system check during .desktop installation

Forwarded: not-needed

the files can also be installed on hurd and kfreebsd
---
 desktop/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/desktop/Makefile b/desktop/Makefile
index b14f0a4..369bb73 100644
--- a/desktop/Makefile
+++ b/desktop/Makefile
@@ -27,7 +27,6 @@ all:
 #
 
 install:
-	if test `uname` = Linux; then \
 		$(INSTALL_DIR) $(BUILDROOT)$(datadir)/applications; \
 		$(INSTALL_DATA) htmldoc.desktop $(BUILDROOT)$(datadir)/applications; \
 		$(INSTALL_DIR) $(BUILDROOT)$(datadir)/mime/packages; \
@@ -38,7 +37,6 @@ install:
 		$(INSTALL_DATA) htmldoc-128.png $(BUILDROOT)$(datadir)/icons/hicolor/128x128/apps/htmldoc.png; \
 		$(INSTALL_DIR) $(BUILDROOT)$(datadir)/icons/hicolor/256x256/apps; \
 		$(INSTALL_DATA) htmldoc-256.png $(BUILDROOT)$(datadir)/icons/hicolor/256x256/apps/htmldoc.png; \
-	fi
 
 
 #
